README.md

```diff
@@ -58,6 +58,46 @@ module ---> dto
 
 OpenAPI schema is auto-generated from the `dtos` and exposed on the `/docs` endpoint.
 
+
+## Starting the bee-api infrastructure
+
+The easiest way to run all the dependencies for the bee-api is to:
+
+- Clone the bee-stack repository https://github.com/i-am-bee/bee-stack
+- Navigate to the bee-stack directory.
+- Run the infrastructure:
+
+```
+./bee-stack.sh clean
+./bee-stack.sh start:infra
+```
+
+- Then navigate back to the bee-api repository.
+- Install packages:
+
+```
+pnpm install
+```
+
+- Run the database seeder:
+
+```
+pnpm mikro-orm seeder:run
```

Comment:
Only for the first time run seeder and in other cases run migrations.

Reply:
Migrations are introduced here: https://github.com/i-am-bee/bee-api/pull/87/files#diff-9a5696f5d0097ae95b8ca91ce055d3a33927d496192ac14c5aa3a5da6c8be9dc

Reply:
Changed to address your first comment. How do the migrations impact this step?